_data/simple-icons.json

```
@@ -5945,6 +5945,11 @@
 		"hex": "DE00A5",
 		"source": "https://www.futurelearn.com"
 	},
+	{
+		"title": "Fyle",
+		"hex": "FF3366",
```

```suggestion
		"hex": "FF2E63",
```

According to their source file on the homepage header, the color should be `#FF2E63`.